mmv1/third_party/terraform/tests/resource_artifact_registry_repository_test.go.erb

```diff
@@ -160,3 +160,135 @@ resource "google_artifact_registry_repository" "test" {
 }
 `, repositoryID)
 }
+
+<% unless version == 'ga' -%>
+
+func TestAccArtifactRegistryRepository_virtual(t *testing.T) {
+	t.Parallel()
+
+        upstreamRepositoryID := fmt.Sprintf("tf-test-%d", randInt(t))
+        repositoryID := fmt.Sprintf("%s-virtual", upstreamRepositoryID)
+
+	vcrTest(t, resource.TestCase{
```

Comment:
The name has to be changed to VcrTest

Reply:
after rebasing main branch

Reply:
Done. Sorry for missing this in the merge.